tests/functional_test.js

```diff
@@ -222,8 +214,7 @@ describe('functional', function () {
                 return;
             }
 
-            var domain = helpers.domainCheck('https://maxcdn.bootstrapcdn.com/');
-            var uri = domain + root + '/' + name;
+            var uri = 'https://maxcdn.bootstrapcdn.com/' + root + '/' + name;
```

Comment:
All right, I'll see what I can keep from these patches, otherwise I'll close the PR.